templates/common/_base/units/machine-config-daemon-firstboot.service.yaml

```
@@ -14,7 +14,7 @@ contents: |
   Type=oneshot
   RemainAfterExit=yes
   # Disable existing repos (if any) so that OS extensions would use embedded RPMs only
-  ExecStartPre=-/usr/bin/sh -c "sed -i 's/enabled=1/enabled=0/' /etc/yum.repos.d/*.repo"
+  ExecStartPre=-/usr/bin/sh -c "sed -i 's/enabled=1/enabled=0/' /etc/yum.repos.d/*.repo; sed -i 's/enabled=0/enabled=1/' /etc/yum.repos.d/rhel-coreos-8-extensions.repo"
```

```suggestion
  ExecStartPre=-/usr/bin/sh -c "sed -i 's/enabled=1/enabled=0/' /etc/yum.repos.d/*.repo; sed -i 's/enabled=0/enabled=1/' /etc/yum.repos.d/coreos-extensions.repo"
```
Removing this requirement might be a bit tricky. In OKD, we boot an FCOS node, and then rebase that to OKD's layered image. The yum repos already exist in the FCOS that is first provisioned, and the rebase won't overwrite file contents in /etc IIUC, so we have to do it here instead.

---

But do we still need to disable the repo files?  Now that OKD is using layering, we're no longer installing packages on each node individually right?

IOW we can just delete this line, correct?

---

oh right. Looks like it can just be removed then.